min-image/runtimes/python/server.py

```diff
@@ -28,19 +28,44 @@ def web_server():
     import f
 
     class SockFileHandler(tornado.web.RequestHandler):
-        def post(self):
+        # TODO: we should consider how are the different requests used in the context of different applications and functions
+        # and consider what does the validations should look like for example, should we allow POST requests with no payload etc.
+        def handle_request(self):
             try:
                 data = self.request.body
-                try :
-                    event = json.loads(data)
+                try:
+                    event = json.loads(data) if data else None # parse the data if its there, if not, set to None
```

Comment:
I think the code is pretty clear, and the comment actually makes it harder to read